Snakefile

```diff
@@ -28,53 +30,85 @@ rule acquire_data:
         "config/ncbi_datasets_env.yaml"
     shell:
         """
-        datasets download virus genome taxon "Dengue Virus" --filename {output.zip} &> {log}
-        datasets summary virus genome taxon "Dengue Virus" --released-after 2000-01-01 --as-json-lines | dataformat tsv virus-genome > {output.metadata} 2>> {log}
+        datasets download virus genome taxon "Dengue Virus" --filename {output.zip} --released-after {params.date} &> {log}
+        datasets summary virus genome taxon "Dengue Virus" --released-after {params.date} --as-json-lines | dataformat tsv virus-genome > {output.metadata} 2>> {log}
         
         # Unzip the downloaded data
         unzip -d data {output.zip}
```

Comment:
When running the pipeline I received this message:
```bash
[Tue Mar  5 14:09:56 2024]
Job 12: Acquiring Genomic Data and Metadata from NCBI database
Reason: Missing output files: data/ncbi_dataset/data/genomic.fna, data/metadata.tsv

Activating conda environment: .snakemake/conda/d01cf8d4eb456008f3bfe89796b5fc97_
Archive:  data/genbank_data.zip
replace data/README.md? [y]es, [n]o, [A]ll, [N]one, [r]ename:
```

My suggestion to avoid this would be to pass the `-o` (overwite) option to `unzip`, which force overwrites (we want to avoid any user interaction in the workflows).
```suggestion
        unzip -o -d data {output.zip}
```

Reply:
sounds good